pkg/sfu/publisher.go

```diff
@@ -201,12 +218,42 @@ func (p *Publisher) AddICECandidate(candidate webrtc.ICECandidateInit) error {
 	return nil
 }
 
-func (p *Publisher) relayReports() {
+func (p *Publisher) createRelayTrack(track *webrtc.TrackRemote, receiver Receiver, rp *relay.Peer) error {
+	codec := track.Codec()
+	downTrack, err := NewDownTrack(webrtc.RTPCodecCapability{
+		MimeType:     codec.MimeType,
+		ClockRate:    codec.ClockRate,
+		Channels:     codec.Channels,
+		SDPFmtpLine:  codec.SDPFmtpLine,
+		RTCPFeedback: []webrtc.RTCPFeedback{{"nack", ""}, {"nack", "pli"}},
+	}, receiver, p.cfg.BufferFactory, p.id, p.cfg.Router.MaxPacketTrack)
+	if err != nil {
+		Logger.V(1).Error(err, "Create Relay downtrack err", "peer_id", p.id)
+		return err
+	}
+
+	sdr, err := rp.Send(receiver.(*WebRTCReceiver).receiver, track, downTrack)
+	if err != nil {
+		Logger.V(1).Error(err, "Relay err", "peer_id", p.id)
```

Comment:
```suggestion
		Logger.V(1).Error(err, "Relaying track.", "peer_id", p.id)
```
Also include track id?